src/watchdog/__init__.py

```diff
@@ -185,6 +191,42 @@ def get_aws_security_credentials_from_ecs(uri):
     return None
 
 
+def get_aws_security_credentials_from_webidentity(role_arn, token_file):
+    try:
+        with open(token_file, 'r') as f:
+            token = f.read()
+    except Exception as e:
+        if is_fatal:
```

Comment:
`is_fatal` is not defined

Reply:
Just pass `is_fatal=False` to the func is OK.

Reply:
I removed `is_fatal` altogether and now just print the error (as the other methods in the watchdog also don't have it).